command/vault/secrets_import.go

```diff
@@ -79,14 +79,14 @@ func SecretsImport(c *cli.Context) error {
 			if err != nil {
```

Comment:
this is a breaking, unrelated change to your PR title, please open a new PR for discussion on this